src/main/java/com/starbucks/model/Order.java

```diff
@@ -21,17 +20,19 @@ public class Order {
     @GeneratedValue(generator = "UUID")
     private UUID id;
 
-    @Column(name = "time", nullable = false)
+    @Column(name = "time")
     private Integer time;
-    @Column(name = "beverages", nullable = false)
+
+    @Column(name = "beverages")
     private String beverages;
-    @Column(name = "size", nullable = false)
+
+    @Column(name = "size")
     private String size;
-    @Column(name = "sugar", nullable = false)
-    private Integer sugar;
 
+    @Column(name = "sugar")
+    private Integer sugar;
```

Comment:
can it be nullable? If not use int instead of Integer. It's about all Integers, Doubles and other packages for primitives.